### ISSUE TITLE
Does not work on Fedora 26

### ISSUE THREAD

**Issue by ziggy42** [mpiannucci/gnome-shell-extended-gestures#16]
I'm using `Fedora 26` with `GNOME 3.24.3` and `wayland` on a `DELL XPS 13 (2013)`. `libinput` is also installed.
Gestures don't work for either 3 and 4 fingers.

**Comment by ziggy42** [comment]
I imagine the problem is not in the extension, since with `libinput debug-events` I cannot see events for 3/4 fingers scrolling

**Comment by corebots** [comment]
I can confirm that after installing the extension on F26 it doesn't work.

**Comment by mpiannucci** [comment]
It is very hardware dependent. @ziggy42 which model XPS is the 2013 one? It might be too old or something becasue I know the 2015+ ones work. 

@corebots which laptop have you tested on?

**Comment by ziggy42** [comment]
@mpiannucci It's the L322X

**Comment by corebots** [comment]
@mpiannucci I'm on a desktop box, with a wacom touch tablet...

**Comment by mpiannucci** [comment]
Out of curiosity does the default four fingers to change workspace gesture that is built into gnome work? If that does not work then I am afraid the extension won't work for you either becasue it uses the same driver functionality

**Comment by ziggy42** [comment]
I didn't even know that was built-in :sweat_smile:

**Comment by corebots** [comment]
nope, the default gestures don't work for me either. the only what works for my setup is https://github.com/bulletmark/libinput-gestures I was hoping you use the same, just with a GUI. Is there a possibility to implement that code please?

**Comment by mpiannucci** [comment]
No, Im sorry... this extension is made to run using the same methods as gnome so that it is guaranteed to work if the default gnome handling works on your system. 

I am curious why your touchpad is not picked up by a gnome wayland session though.

**Comment by mpiannucci** [comment]
Hold on.... is `libinput-gestures` installed? It will not work while that is installed.
